css-canvas-medical/index.html

```diff
@@ -0,0 +1,103 @@
+        <button class="contact">Contact Us</button>
+      </div>
+    </div>
+    <hr>
+    <div class="row hero-row">
+      <div class="column-full hero-styles">
+        <div class="overlay"></div>
+        <h1 class="emr">The EMR Reimagined for Primary Care</h1>
+        <p class="emr-description">
+          Can't remember the last time you talked with a patient without focusing
+          on the EMR? Canvas lets you rediscover joy in the work you do. Capture
+          meaningful data, identify gaps in care, and move seamlessly between
+          tasks within the platform. You stay present with patients while your
+          whole care team stays up-to-speed. Be a better doctor with Canvas,
+          designed for primary care providers with the help of primary care
+          providers.</p>
+        <button class="get-started">Get Started</button>
+      </div>
+    </div>
+    <div class="row prefer-canvas">
+      <div class="column-full prefer-styles">
+        <h1 class="clinicians">Clinicians Prefer Canvas</h1>
+        <p class="clinician-description">Physician-owned primary care practices
+          across the country have <br/>
+          already made Canvas the fabric of their care.</p>
+      </div>
+      <div class="row location">
+        <div class="column-half locations">
+          <p>Cascade Family Practice, Portland OR</p>
+          <p>Temescal Creek Medicine, Oakland CA</p>
+          <p>San Juan Healthcare, Friday Harbor WA</p>
+        </div>
+
+        <div class="column-half locations">
+          <p>Ward Medical Clinic, Bonifay FL</p>
+          <p>Caribou Medical, Towanda PA</p>
+          <p>Bay Tree Family Care, Red Bay AL</p>
+        </div>
+      </div>
+    </div>
+    <div class="row learn-more">
+      <div class="column-half learn">
+        <h1>Ready to learn more?</h1>
+        <p>Schedule a private meeting now</p>
+      </div>
+      <div class="column-half started">
+        <button class="start">Get Started</button>
+      </div>
+    </div>
+    <div class="row information">
+      <div class="column-half info-styles">
+        <img class="logo" src="images/logo-white.png">
+        <p class="address">2307 Irving Street, Suite 228</p>
+        <p class="address">San Francisco CA 94122</p>
+        <p class="address">(800) 370-1416</p>
+        <p class="address">info@canvasmedical.com</p>
+      </div>
+      <div class="column-quarter quarter-styles">
+        <h4>COMPANY</h4>
+        <p>About Us</p>
+        <p>Blog</p>
+      </div>
+      <div class="column-quarter quarter-styles">
+        <h4>PRODUCT</h4>
+        <p>Product</p>
+        <p>Implementation</p>
+        <p>For Health Plans</p>
+        <p>Security</p>
+      </div>
+    </div>
+    <footer class="row foot">
+      <div class="column-quarter footer-text">
+        <p>Privacy &amp; Terms</p>
+        <p>Contact Us</p>
+      </div>
+      <div class="column-three-fourth"></div>
```

Comment:
You should never have empty columns to create space. You can just delete this element.